docs/source/changelog.rst

```diff
@@ -7,7 +7,7 @@ Changelog
 Array
 +++++
 
--
+- Reimplemented ``argtopk`` to make it release the GIL (:pr:`3596`) `Guido Imperiale`
```

Comment:
You'll want to terminate your name here with an underscore like 

    `Guido Imperiale`_